### ISSUE TITLE
Debugging documentation

### ISSUE THREAD

**Issue by AqlaSolutions** [AsynkronIT/protoactor-dotnet#2017]
I want to debug an app that uses ProtoActor with Consul. It took me long time to understand that I need to increase `HeartbeatExpiration` so that app will not shutdown by itself. Are there any caveats like this? Could you please add it to documentation?

**Comment by rogeralsing** [comment]
You are correct, in the latest previews there is also a .WithDisableHeartbeatExpiration() on cluster config for this reason.

We are planning to create some reference setups for both local and prod environments and explain the reason for each setting there.

**Comment by AqlaSolutions** [comment]
Do I need to also change anything in Consul provider options in this case?

**Comment by AqlaSolutions** [comment]
I changed "deregister critical" to 1h but this still seems not enough. Sometimes it happens that `UpdateClusterTopology` is called without some node listed in the members list so this node gets blocked. In the consul output I see only `Check missed TTL, is now critical` messages but nothing about service deregistration.

**Comment by rogeralsing** [comment]
Closing due to inactivity. Reopen if still relevant.
